.github/workflows/jmh-benchmarks.yml

```diff
@@ -28,8 +28,8 @@ on:
         description: 'The branch name'
         required: true
       spark_version:
-        description: 'The spark project version to use, such as iceberg-spark-3.5'
-        default: 'iceberg-spark-3.5'
+        description: 'The spark project version to use, such as iceberg-spark-4.0'
+        default: 'iceberg-spark-4.0'
```

Comment:
I think it would be better if we hold off on updating this to use 4.0 until the release is official? We can update it alongside the default spark version update when the Spark 4.0 is actually released.  Until then I think it makes sense for JMH benchmarks to just run against 3.5

Reply:
Changed